test/one/nio/serial/JsonReaderTest.java

```diff
@@ -108,10 +108,56 @@ public void nullableFields() throws IOException, ClassNotFoundException {
         Assert.assertTrue(x.set instanceof Set && x.set.isEmpty());
     }
 
+    @Test
```

Comment:
`jsonReaderWriter()` test above doesn't pass anymore due to very big `long` serialized as string. Could you please fix it?

Reply:
I've since made some more changes to the code and tests. I'll update the pull request.